Libraries/NativeAnimation/RCTAnimationUtils.m

```diff
@@ -93,3 +93,17 @@ CGFloat RCTDegreesToRadians(CGFloat degrees)
 {
   return degrees / 180.0 * M_PI;
 }
+
+#if TARGET_IPHONE_SIMULATOR
+// Based on https://stackoverflow.com/a/13307674
+float UIAnimationDragCoefficient(void);
+#endif
+
+CGFloat RCTAnimationDragCoefficient()
+{
+#if TARGET_IPHONE_SIMULATOR
+  return (CGFloat)UIAnimationDragCoefficient();
```

Comment:
Based on this POP code, maybe we need to guard against 0 for some reason?

https://github.com/facebook/pop/blob/master/pop/POPAnimationExtras.mm#L37

Reply:
Oh that's strange. Thanks for investigating this!